distribution/openrct2.d.ts

```diff
@@ -1140,24 +1318,47 @@ declare global {
 
     interface RideSetSettingArgs extends GameActionArgs {
         ride: number;
-        setting: number; // see RideSetSetting in openrct2/actions/RideSetSettingAction.h
+        /** @see RideSetSetting in
+         * [src/openrct2/actions/RideSetSettingAction.h](../src/openrct2/actions/RideSetSettingAction.h)
```

Comment:
Nitpicky, but seems more consistent with the rest:

```suggestion
        /** 
         * @see RideSetSetting in [src/openrct2/actions/RideSetSettingAction.h](../src/openrct2/actions/RideSetSettingAction.h)
```